scripts/build.sh

```diff
@@ -135,7 +143,7 @@ usage() {
 }
 
 # Setup build flags (release, debug, sanitizer, ...)
-while getopts hb:s:e:r:dS:c:C flag
+while getopts hb:s:e:r:dS:c:C:a flag
```

Comment:
Remove colon after `C`, it doesn't take an argument.

Reply:
Oops, removed the colon.